package.json

```diff
@@ -17,10 +17,19 @@
   },
   "homepage": "https://github.com/cf-lms/lms#readme",
   "devDependencies": {
+    "babel-core": "^6.3.17",
+    "babel-loader": "^6.2.0",
+    "babel-preset-react": "^6.3.13",
     "chai": "^3.4.1",
-    "mocha": "^2.3.4"
```

Comment:
these are not accidental changed. They are to keep the devDependencies in alphabetical order.